src/routes.js

```diff
@@ -805,8 +827,16 @@ const routesByName = new Map();
       : false)
   );
 
-
-
+  [
+    'NewSubmission',
+    'NewDraftSubmission',
+    'EditSubmission',
+    'FormPreview',
+    'DraftFormPreview'
```

Comment:
At least when naming components, we tend to list the resource first, then the action or descriptor after, e.g., `EntityUpdate` or `ProjectOverview`. What do you think about doing something similar here?

Also, in the past, we've named things `FormDraft` rather than `DraftForm`. I think either one makes sense, but I think consistency is helpful.

```suggestion
    'SubmissionNew',
    'DraftSubmissionNew',
    'SubmissionEdit',
    'FormPreview',
    'FormDraftPreview'
```